other/Settings.php

```diff
@@ -225,6 +231,20 @@
  */
 $tasksdir = $sourcedir . '/tasks';
 
+/**
+ * Path to the cache API directory.
+ *
+ * @var string
+ */
+$cacheAPIdir = $sourcedir . '/cache';
+
+/**
+ * Path to the cache implementations.
+ *
+ * @var string
+ */
+$cacheAPIImplementationdir = $cacheAPIdir . '/implementations';
+
```

Comment:
Overall, do we need to build a $cacheAPIdir and $cacheAPIImplementationdir  for any reason?  Why not just build off $sourcedir?  I personally do not believe we need to do that, same with $tasksdir but I wasn't around for that implantation.  Just adds complexity to the code.

Reply:
I'm not a fan of using $sourcedir .'/harcoded_dir/hardcoded_dir';

It's easier to reference an already build global variable than building your own. Mod authors will end up building their own anyway, might as well provide a uniform and consistent across modifications variable ;)

Reply:
Went with a half way approach, kept cacheAPIdir, deleted the other one.